code/game/objects/machinery/buttons.dm

```diff
@@ -249,33 +249,51 @@
 	else
 		icon_state = "doorctrl0"
 
-/obj/machinery/button/valhalla_button
-	name = "Xeno spawner"
+/obj/machinery/button/valhalla
 	resistance_flags = INDESTRUCTIBLE
-	///The xeno created by the spawner
-	var/mob/living/xeno
-	///What spawner is linked with this spawner
-	var/link = CLOSE
+	///The mob created by the spawner
+	var/mob/living/carbon/human/linked
+	///What spawner landmark is linked with this spawner (this has to be matching with the landmark)
+	var/link
+
+/obj/machinery/button/valhalla/Destroy()
+	linked = null
+	return ..()
+
+/obj/machinery/button/valhalla/marine_button
+	name = "Xeno spawner"
 
-/obj/machinery/button/valhalla_button/attack_hand(mob/living/user)
+/obj/machinery/button/valhalla/marine_button/attack_hand(mob/living/user)
```

Comment:
*marine button"
spawns xeno

proof of the cabal

Reply:
Marine button as in button used by marines *shrug